src/structures/APIMessage.js

```diff
@@ -192,7 +193,7 @@ class APIMessage {
       tts,
       nonce,
       embed: this.options.embed === null ? null : embeds[0],
-      embeds,
+      embeds: isWebhook ? embeds : undefined,
```

Comment:
This needs isInteraction as well, kinda starting to feel like isWebhook should just include isInteraction

Reply:
As per above: This currently breaks `interaction.reply({ embeds: [{description: "1"}, {description: "2"}] });`